lib/SPIRV/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ add_llvm_library(LLVMSPIRVLib
   SPIRVLowerConstExpr.cpp
   SPIRVLowerSPIRBlocks.cpp
   SPIRVLowerMemmove.cpp
+  SPIRVLowerOCLBlocks.cpp
```

Comment:
Could we re-alphabetise this list, I'm sorry I'm a bit of a pedant :)

Reply:
No problem :)